src/app/widget/widget.component.ts

```diff
@@ -1,10 +1,16 @@
 import { Component } from '@angular/core';
+import {ExportService} from "../services/export-service";
 
 @Component({
   selector: 'app-widget',
   templateUrl: './widget.component.html',
-  styleUrls: ['./widget.component.css']
+  styleUrls: ['./widget.component.css'],
+  providers: [ExportService]
```

Comment:
when you provide your service in that way :

@Injectable({
  providedIn: 'root'
})


you don't need to provide it here